go/nbs/store.go

```diff
@@ -163,7 +165,7 @@ func (nbs *NomsBlockStore) GetMany(hashes []hash.Hash) []chunks.Chunk {
 		tables = nbs.tables
 
 		if nbs.mt != nil {
-			remaining = nbs.mt.getMany(reqs)
+			remaining = nbs.mt.getMany(reqs, &sync.WaitGroup{})
```

Comment:
fair. will wait on it. it'll be noop. and it'll go away soon with channel getMany